src/dhtproto/node/request/Listen.d

```diff
@@ -91,7 +93,15 @@ public abstract scope class Listen : SingleChannel
             }
             finally
             {
-                this.reader.fiber.epoll.unregister(disconnect_detector);
+                static if (hasMethod!(typeof(this.reader.fiber.epoll),
```

Comment:
One last point: maybe add a comment (containing the magic keyword "deprecated") about removing this code in the future.

Reply:
Updated.